janus_server/src/hpke.rs

```diff
@@ -469,4 +747,175 @@ mod tests {
             .open(&ciphertext, b"wrong associated data")
             .unwrap_err();
     }
+
+    fn round_trip_check<KEM: hpke::Kem, KDF: hpke::kdf::Kdf, AEAD: hpke::aead::Aead>(
+        task_id: TaskId,
```

Comment:
nit: why accept `task_id`? it looks like it's only used in one place, and it doesn't need to match up between calls to this function AFAICT. We could just call `TaskId::random()` (or a "zero" task ID) directly.

Similarly, instead of passing `&mut rng`, drop the parameter and call `thread_rng()` directly as needed?